lib/main.js

```diff
@@ -78,6 +79,19 @@ var init = exports.init = function(host, options) {
                 callback(null, data);
             });
         },
+        stopBuild: function(jobname, buildNumber, callback) {
```

Comment:
Could this match the format of other commands and variable using the `_` to denote spaces? So `stopBuild` → `stop_build`? Obviously up to @jansepar, but it looks weird to mix formats to me.